Dota - Flaky specs

## Describe the bug
We have some flaky specs at the moment

## To Reproduce
Describe a way to reproduce your bug. To get the Faker version, run `Faker::VERSION`.

Use the reproduction script below to reproduce the issue:

Run CI, or look at this previous failed run:
https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769

## Expected behavior
No flaky tests!

## Additional context

Here's an example:
```
===============================================================================
Failure: test_player(TestFakerDota):
  </\w+/> was expected to be =~
  <"マテリス">.
/home/runner/work/faker/faker/test/faker/games/test_faker_dota.rb:33:in `test_player'
     30:   end
     31: 
     32:   def test_player
  => 33:     assert_match(/\w+/, @tester.player)
     34:   end
     3[5](https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769#step:6:6): 
     3[6](https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769#step:6:7):   def test_heroes_quotes
===============================================================================
Finished in 14.20[7](https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769#step:6:8)26965 seconds.
Stopped processing SimpleCov as a previous error not related to SimpleCov has been detected
2125 tests, 273573 assertions, 1 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
JSON Coverage report generated for Unit Tests to /home/runner/work/faker/faker/coverage.         4629 / 46[8](https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769#step:6:9)0 LOC ([9](https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769#step:6:10)8.91%) covered.
rake aborted!
Command failed with status (1)
/opt/hostedtoolcache/Ruby/3.0.6/x64/bin/bundle:[23](https://github.com/faker-ruby/faker/actions/runs/5481690752/jobs/9986269930?pr=2769#step:6:24):in `load'
/opt/hostedtoolcache/Ruby/3.0.6/x64/bin/bundle:23:in `<main>'
Tasks: TOP => test
(See full trace by running task with --trace)
Error: Process completed with exit code 1.
```

--- a/test/faker/games/test_faker_dota.rb
+++ b/test/faker/games/test_faker_dota.rb
@@ -30,7 +30,7 @@ class TestFakerDota < Test::Unit::TestCase
   end
 
   def test_player
-    assert_match(/\w+/, @tester.player)
+    assert_match( /[\p{Katakana}\p{Han}\p\w]+/, @tester.player)
   end
 
   def test_heroes_quotes

--- a/test/faker/games/test_faker_dota.rb
+++ b/test/faker/games/test_faker_dota.rb
@@ -30,7 +30,7 @@ class TestFakerDota < Test::Unit::TestCase
   end
 
   def test_player
-    assert_match(/[\p{Katakana}\p{Han}\p\w]+/, @tester.player)
+    assert_match(/[\p{Katakana}\p{Han}\w]+/, @tester.player)
   end
 
   def test_heroes_quotes

--- a/test/faker/games/test_faker_dota.rb
+++ b/test/faker/games/test_faker_dota.rb
@@ -30,7 +30,7 @@ class TestFakerDota < Test::Unit::TestCase
   end
 
   def test_player
-    assert_match( /[\p{Katakana}\p{Han}\p\w]+/, @tester.player)
+    assert_match(/[\p{Katakana}\p{Han}\p\w]+/, @tester.player)
   end
 
   def test_heroes_quotes

